cranelift/frontend/src/frontend.rs

```diff
@@ -206,6 +206,11 @@ impl<'a> FunctionBuilder<'a> {
         }
     }
 
+    /// Get the block that this builder is currently at.
+    pub fn current_block(&self) -> Option<Block> {
+        self.position.expand()
+    }
```

Comment:
Right now, all users of this use it to do `builder.func.layout.insert_block_after`. The layout is public, though it is somewhat low-level, and it's desirable to avoid having lots of code directly interacting with it. Could you add a `insert_block_after` method to `FunctionBuilder`?

Reply:
Good idea -- done!